### ISSUE TITLE
Webhooks return 400

### ISSUE THREAD

**Issue by FrostyX** [fedora-copr/copr#2416]
(Custom) Webhooks seem to have some issues after the last release. Originally reported on IRC

> <xsnrg>  Are there any known problems with copr webhooks, or recent
>    changes I might not be aware of?
> <xsnrg>  I have been getting 400s recently on webhooks that have been
 >   working for a very long time

Looking at the logs, I can see 

```
2022-12-02 16:03:05,527 [ERROR][/usr/share/copr/coprs_frontend/coprs/error_handlers.py:42|error_handlers:handle_error][ANON] Response error: 400 The browser (or proxy) sent a request that this server could not understand.
2022-12-02 16:03:05,532 [INFO][/usr/share/copr/coprs_frontend/coprs/views/webhooks_ns/__init__.py:25|__init__:after_request][ANON] Webhook (400) data: b''
```

**Comment by praiskup** [comment]
I don't think this is related to our code change, but to some flask change.  I think that json mime header in the POST request is newly needed.

**Comment by praiskup** [comment]
Please retry with `curl -H "Content-Type: application/json"`.

**Comment by xsnrg** [comment]
I tried with -H "Content-Type: application/json", which failed, so I also tried using curl -X POST -json which should add both the Content-Type header as well as the accept header. This also did not work.

**Comment by praiskup** [comment]
Nah, thank you for the feedback.  I just tested now:
```
$ curl -H "Content-Type: application/json" --data '{}' -X POST https://copr.fedorainfracloud.org/webhooks/custom/5893/<UUID_HASH>/copr-cli/
5079464
```
The [build has started](https://copr.fedorainfracloud.org/coprs/g/copr/copr/build/5079464/).

**Comment by xsnrg** [comment]
Confirmed, that works for me as well. I guess what is needed is a web page update for the examples here: https://docs.pagure.org/copr.copr/user_documentation.html#webhooks
